From 2f888c89227cb68e9e854a1078224b398c58b0dc Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Wed, 14 Oct 2015 19:11:42 +0800
Subject: [PATCH 0930/1691] MLK-11711-01 ARM: dts: imx7d-12x12-lpddr3-arm2-sai:
 enable sai<->wm8958 sound card support in lpsr mode

commit 2f888c89227cb68e9e854a1078224b398c58b0dc from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

enable sai<->wm8958 sound card support in lpsr mode

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts
index c2c0da5..719efca 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-sai.dts
@@ -21,12 +21,15 @@
 };
 
 &iomuxc {
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_hog_1>;
+	pinctrl-1 = <&pinctrl_hog_1>;
 };
 
 &sai1 {
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_sai1>;
+	pinctrl-1 = <&pinctrl_sai1>;
 	status = "okay";
 };
 
-- 
1.9.1

